dp/cloud/python/magma/configuration_controller/response_processor/response_db_processor.py

```diff
@@ -72,17 +73,14 @@ def process_response(self, db_requests: List[DBRequest], response: requests.Resp
             return
 
     def _populate_grant_states_map(self, session):
-        self.grant_states_map = {
-            GrantStates.GRANTED.value: session.query(DBGrantState).filter(
-                DBGrantState.name == GrantStates.GRANTED.value,
-            ).scalar(),
-            GrantStates.AUTHORIZED.value: session.query(DBGrantState).filter(
+        grant_states = session.query(DBGrantState).filter(
```

Comment:
why not just `session.query(DBGrantState).all()`?

Reply:
true, didn't notice these were the only possible states